src/web/src/components/SearchInput/AuthorSearchInput.tsx

```diff
@@ -33,6 +33,8 @@ const AuthorSearchInput = () => {
   return (
     <>
       <input
+        // eslint-disable-next-line jsx-a11y/no-autofocus
+        autoFocus
         className={classes.input}
         list="search-suggestions"
```

Comment:
While you are, can you remove this `list` attribute and the `data-list` field below? They're used for input suggestions but we don't have any.

Reply:
sure @DukeManh